scss/variables/_buttons.scss

```diff
@@ -0,0 +1,32 @@
+.button {
+  color: #fff;
+  &__main {
+		display: inline-block;
+		font-family: Aeonik;
+		font-size: 22px;
+		font-weight: 500;
+		font-style: normal;
+    padding: 15px 29px;
+    border-radius: 2px;
+    background: #010101;
+    color: #fff;
+    position: relative;
+    display: flex;
+    align-items: center;
+    gap: 15px;
+    transition: all 0.3s ease;
+
+    &:hover {
+      background-color: #fff;
+      color: black;
+      border: 1px solid #000;
```

Comment:
while hovering button should be same size
instead it's made bigger, because of border-width
![image](https://github.com/zahurnyi/school/assets/96112534/a860ff8d-8ea0-4002-b363-70715592d272)